docs/examples/carousel/auto-height.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <el-carousel height="auto" :autoplay="false">
```

Comment:
```suggestion
  <el-carousel height="auto" autoplay>
```

Set different heights for `carousel-item`